$MirOS: ports/net/irssi/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:48:26 tg Exp $
--- configure.in.orig	Sun Dec 11 10:40:59 2005
+++ configure.in	Wed Apr 18 22:36:47 2007
@@ -17,6 +17,7 @@ AC_PROG_CPP
 AC_STDC_HEADERS
 AC_ARG_PROGRAM
 AM_PROG_LIBTOOL
+AC_SUBST(AR)
 
 AC_PATH_PROG(sedpath, sed)
 AC_PATH_PROG(perlpath, perl)
